docker_files/home/lisk/run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+confd -backend env -onetime
+node app.js
```

Comment:
Normally I noticed `entrypoint.sh` as startup file in different docker images, should we not follow that naming convention.